### DISCUSSION TITLE
About Integrating itwin

### DISCUSSION THREAD

**Discussion by saichandu208** [iTwin/community#5] [Q&A] [answered]
Can i get any  pdf or  video  about how to do  integration of itwin with any other applications.

**Comment by aruniverse** [comment]
Hi,

Can you be a little more specific about what you are trying to accomplish? 
Here's a [playlist](https://youtube.com/playlist?list=PL6YCKeNfXXd9GjLi9ICRb0pACkR0OHTEH) from our DevConference last year where we go over various parts of the iTwin Platform which might be a good starting point for you

**Comment by saichandu208** [comment]
Hi 
I was going through Accreditation course , I was getting an error even after providing  .env variables, Here i was attaching the error  screen shot, please help in that case. 
![env error](https://github.com/iTwin/community/assets/83529432/6c09493e-a4b2-40a6-885f-ab3cf4a67ce3)
 
![env2](https://github.com/iTwin/community/assets/83529432/d55bb5a6-01d8-4bae-8284-4effb4d35c1b)

**Reply by aruniverse** [reply, marked as answer]
did you restart your application like the error said after adding to the .env file?

**Reply by saichandu208** [reply]
Thanks, now it's working.
